src/main/java/com/myMoneyTracker/model/category/Category.java

```diff
@@ -32,6 +32,10 @@
     @JoinColumn(name = "user_id")
     private AppUser user;
 
+    @NotNull
+    @Length(min = 7, max = 7)
+    private String colour = "#FFF0F5";
```

Comment:
No. If the user wants to group by colour multiple categories by associating them to a bigger category then the colour doesn't have to be unique. For the time, let's keep this as it is